.github/workflows/builder_nodejs_slsa3.yml

```diff
@@ -60,29 +48,37 @@ on:
         required: false
         type: string
 
-      dist-tag:
-        description: "The package dist-tag to attach. See `npm help dist-tag` for more information."
-        required: false
-        type: string
-        default: "latest"
-
-    secrets:
-      node-auth-token:
-        description: "The npm registry auth token used to publish the package."
-        required: true
-
     outputs:
-      artifact-name:
+      package-name:
+        description: "The file name of the package tarball in the upload artifact."
+        # NOTE: This is an "untrusted" value returned from the build.
+        value: ${{ fromJSON(jobs.slsa-run.outputs.build-artifacts-outputs).package-filename }}
+
+      package-download-name:
```

Comment:
wondering if we want to make this package a folder, like for attestations. If we update the workflow to support multiple packages in the future, that could be useful? That may mean outputting information about package and their corresponding attestation, like Goreleaser does. We may be able to use the slsa-layout... or a similar format.

Note: not a blocker, just curious about your thoughts.

Reply:
I think it's more likely that we'll just have folks call the workflow multiple times if they need to build multiple packages.

Related to [Matrix Strategies](https://docs.google.com/document/d/1pA75l1ugpe5clT2eNceVZRdtW7A7hIuCzoze7FCWmFQ/edit?resourcekey=0-4DuG7lPMreQWbJO7rS8Icw#heading=h.zlp53mm5rv1) section in the design doc.